CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# Contribution Guidelines
+
+Any change to this repository is always welcomed, and greatly appreciated. You can add new entries, or modify/remove existing entries if they are changed/discontinued. You are also allowed to add/modify categories as you see fit.
+
+Please ensure your pull request adheres to the following guidelines:
+
+-   Search previous suggestions before making a new one, as yours may be a duplicate.
+-   Make an individual pull request for each suggestion.
+-   If adding/modifying an entry, use the following format:
+    `[NAME](LINK) - DESCRIPTION.`
```

Comment:
Instead of proposing which format to follow for new items, I would instead say something along the lines of "keep a consistent format with the rest of the document". If the document style changes, new entries will be entered differently as well.

Reply:
I honestly doubt this format would change, especially if we decide to adhere to the same format of awesome lists. What I could do is re-word is so it sounds more like a suggestion than a strict guideline